processor/workers.go

```diff
@@ -486,6 +488,15 @@ func CountStats(fileJob *FileJob) {
 		if fileJob.Content[index] == '\n' || index >= endPoint {
 			fileJob.Lines++
 
+			if MinifiedGenerated {
+				if avgLineByteCount == 0 {
+					avgLineByteCount = index - lineOffset
+				} else {
+					avgLineByteCount = (avgLineByteCount + (index - lineOffset)) / 2
```

Comment:
This isn't a real average - it's more of a rolling average weighted towards lines at the end of the file.

This doesn't even seem like it's needed. Just stick `avgLineByteCount := len(fileJob.Content) / fileJob.Lines` after the state machine.

Reply:
Ah the old not notice whats going on when you are doing it. Good point.

Reply:
The weighted towards the end was useful since some JS files follow that pattern but this might still work.

Reply:
Tests pass so all good.